### DISCUSSION TITLE
Module 'react-native-gesture-handler/ReanimatedSwipeable' cannot be found

### DISCUSSION THREAD

**Discussion by tba-devel** [software-mansion/react-native-gesture-handler#3132] [Q&A] [answered]
I am trying to implement a swipeable list with the new **Reanimated Swipeable component** but I got 

> Cannot find module 'react-native-gesture-handler/ReanimatedSwipeable' or its corresponding type declarations.ts(2307) 

I installed all the dependencies stated in the website but still I don't know what I am missing, if someone did experience the same, how did you solve it?

**Comment by latekvo** [comment, marked as answer]
Hi @tba-devel.

The `ReanimatedSwipeable` component was introduced in `RNGH` version `2.18.1`, are you certain the version you're using is equal or higher?

If that doesn't help, could you please share with us an example of this failing?

**Reply by tba-devel** [reply]
Well it's working now, I was using RNGH v2.16.1 and updated to 2.18.1. I'm certain that piece of information you gave would be useful also at the [website](https://docs.swmansion.com/react-native-gesture-handler/docs/components/reanimated_swipeable) since there's no mention of it whatsoever. Thanks by the way!
